tests/test_gp_interp.py

```diff
@@ -47,18 +47,12 @@ def get_correlation_length_matrix(correlation_length, g1, g2):
     :param correlation_length: Correlation lenght of the kernel.
     :param g1, g2:             Shear applied to isotropic kernel.
     """
-    if abs(g1)>1:
-        g1 = 0
-    if abs(g2)>1:
-        g2 = 0
+    if abs(g1)>1 or abs(g2)>1:
+        raise ValueError('abs value of g1 and g2 must be lower than one')
```

Comment:
Probably don't need this line actually, now that you are using galsim.Shear().  It will raise an appropriate error if g1,g2 are invalid.  (Which also includes when they are both <1, but g1^2 + g2^2 > 1.)